backstop/tests/breadcrumbs.html

```diff
@@ -55,155 +54,324 @@
     <h1>Breadcrumbs</h1>
     <hr />
 
-    <section id="demo-all">
+    <h2>Generic anchor</h2>
+    <section id="demo-anchors">
 
-      <section id="demo-anchors">
+      <nav
+        class="iui-breadcrumbs"
+        aria-label="Breadcrumbs"
+      >
+        <button class="iui-button iui-borderless iui-dropdown">
+          <svg-folder
+            class="iui-button-icon"
+            aria-hidden
+          ></svg-folder>
+          <svg-caret-down-small
+            class="iui-button-icon"
+            aria-hidden
+          ></svg-caret-down-small>
+        </button>
+        <ol class="iui-breadcrumbs-list">
+          <li class="iui-breadcrumbs-item">
+            <a href="#">Root</a>
+          </li>
+          <li
+            class="iui-breadcrumbs-separator"
+            aria-hidden
+          >
+            <svg-chevron-right aria-hidden></svg-chevron-right>
+          </li>
+          <li class="iui-breadcrumbs-item">
+            <span class="iui-breadcrumbs-text">&mldr;</span>
```

Comment:
so now its enough just to apply class to child element of `iui-breadcrumbs-item` no matter of its type, right? Asking for react.

Reply:
`.iui-breadcrumbs-item` will work for `<a>`, `<a href="">`, & `<span>`.